lib/open_project/text_formatting.rb

```diff
@@ -379,7 +379,9 @@ def parse_headings(text, _project, _obj, _attr, _only_path, options)
         item = strip_tags(content).strip
         anchor = item.gsub(%r{[^\w\s\-]}, '').gsub(%r{\s+(\-+\s*)?}, '-')
         @parsed_headings << [level, anchor, item]
-        "<a name=\"#{anchor}\"></a>\n<h#{level} #{attrs}>#{content}<a href=\"##{anchor}\" class=\"wiki-anchor\">&para;</a></h#{level}>"
+        url = full_url anchor
+        next "<a name=\"#{anchor}\"></a>\n<h#{level} #{attrs}>#{content}<a href=\"#{url}\" class=\"wiki-anchor\">&para;</a></h#{level}>" if url
```

Comment:
Except from the (valid) hound warning:

````
next foooooooo if bar?
````

Especially for such long lines this is nasty.

How about

````
if bar?
  next fooooooooo
end

I even had to google the meaning of `next 'foo'`, but that is a different topic ^^